templates/_blocks/index.twig

```diff
@@ -0,0 +1,56 @@
+{# @var \Illuminate\Support\Collection<\craft\elements\Entry> blocks #}
+{% set blocks = blocks is defined ? blocks.collect() : collect([]) %}
+
+{% if blocks.count() %}
+  <div class="{{ cx({
+    'pt-64': blocks.first().type.handle not in ['pageHero', 'callToAction'],
+    'pb-64': blocks.last().type.handle not in ['callToAction'],
+  }) }}">
```

Comment:
There is vertical padding applied to the page builder. 

However, if certain full bleed blocks are at the start / end, that padding will be removed.